AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LookModificationAdded.py

```diff
@@ -172,35 +190,77 @@ def AtomEditorComponents_LookModification_AddedToEntity():
             color_grading_lut_asset.id == look_modification_component.get_component_property_value(
                                           AtomComponentProperties.look_modification('Color Grading LUT')))
 
-        # 10. Enter/Exit game mode.
+        # Cycle through options in the Shaper Type parameter.
+        for shaper_type in SHAPER_TYPE.keys():
+
+            # 10. Set the Shaper Type parameter (from atom_constants.py SHAPER_TYPE, default none)
```

Comment:
Not sure if you need to put "#10" here again could just delete it and leave the rest of the comment, since it is assumed that it is already part of 10 from the number above. Minor annoyance though, just slightly confusing